cmd/podman/images/pull.go

```diff
@@ -33,7 +34,7 @@ var (
 	// Command: podman pull
 	pullCmd = &cobra.Command{
 		Use:               "pull [options] IMAGE",
```

Comment:
You have to add this, this will fix the failing test.
Also please update the synopsis in the man page.
```suggestion
		Use:               "pull [options] IMAGE [IMAGE...]",
```